packages/react-components/react-rating-preview/stories/Rating/index.stories.tsx

```diff
@@ -6,7 +6,6 @@ import bestPracticesMd from './RatingBestPractices.md';
 export { Default } from './RatingDefault.stories';
```

Comment:
It would be good to add a "Controlled Value" story showing how to listen for `onChange` and set the `value` prop.